sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -46,7 +46,7 @@ SUBROUTINE ALLOCATE_ALL()
       integer ierr,jsx,jex
       integer i,j,l,k
 ! Allocate arrays
-      allocate(u(im,jsta_2l:jend_2u,lm))
+      allocate(u(im+1,jsta_2l:jend_2u,lm))
```

Comment:
Is u index im+1 used in the code? I am curious why only u field has this dimension.

Reply:
@junwang-noaa The arrays U/V are used by rap/hrrr for their special grids. You may see the read interface  INITPOST.F for rap/hrrr. In my last PR for initializing all post internal variables, I accidentally change it.

Reply:
UH is used for other models and it does have im dimensiion